src/lambda/templates/lambdaTemplates.ts

```diff
@@ -43,12 +43,23 @@ export class LambdaTemplates {
     `
     public static readonly GET_POLICY_TEMPLATE = `
     <h1>
-        Policy for <%= FunctionName %>...
+        Policy for <%= FunctionName %> ...
     </h1>
     <p>Policy:
         <pre><%- JSON.stringify(JSON.parse(Policy), null, 4) %></pre>
     </p>
     `
+    public static readonly GET_POLICY_TEMPLATE_ERROR = `
+    <h1>
+        Policy for <%= FunctionName %> ...
```

Comment:
I don't think that there should be a space before the ellipsis.

Reply:
I've changed this back, but I feel that there should be a space separating the function name and ellipses, to help separate the lambda name from surrounding text. Particularly when you see a long lambda name that runs on to `...`, it is more cluttered.